### DISCUSSION TITLE
We are unable to transcribe Italian speakers

### DISCUSSION THREAD

**Discussion by ubeytd** [deepgram/community#820] [General help] [answered]
Hi everyone,

We are developing a product that is using Deepgram's WebSocket service to real-time transcribe meetings. However, we are having a hard time capturing all words; in fact, we are getting almost 1-2 out of 10 words.

Our default options are:

```js
const defaultOptions = {
  model: "nova-2",
  language: "it",
  interim_results: true,
  smart_format: true,
  filler_words: true,
  utterance_end_ms: 3000,
};
```

What options do you suggest to achieve decent performance?

Thanks for your suggestion and help in advance 🙏🏻

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by ubeytd** [comment]
As additional context : To ensure that our tests are distinct from our application and to prevent our application's potential bugs from affecting the test results, we conduct our tests on an external [example repository provided by deepgram](https://github.com/deepgram-starters/nextjs-live-transcription)

**Comment by jpvajda** [comment, marked as answer]
@ubeytd  there is actually a lot to consider in your question here.  😄  

There are a lot of Docs you can reference to work your way through what you need to consider while using our Streaming API Just check out: https://developers.deepgram.com/docs/stt-streaming-feature-overview. 

I'd suggest focusing on the result processing Docs as this is where you can get the most optimization benefits. 

Also we have an AI powered BOT on our Docs site which can help you answer questions, Just click on "Ask AI" and you can prompt it like ChatGPT.

Finally if this doesn't get you the answers you seek, trying joining our DIscord Community to connect with other Developers who face similar challenges. 

https://discord.com/invite/deepgram
